packages/server/test/e2e/7_record_spec.coffee

```diff
@@ -1371,3 +1371,186 @@ describe "e2e record", ->
             snapshot: true
             expectedExitCode: 0
           })
+
+      describe "free plan - nearing tests limit", ->
+        routes = defaultRoutes.slice()
+        routes[0] = {
+          method: "post"
+          url: "/runs"
+          req: "postRunRequest@2.1.0",
+          res: (req, res) -> res.status(200).json({
+            runId
+            groupId
+            machineId
+            runUrl
+            warnings: [{
+              code: "FREE_PLAN_NEARING_MONTHLY_TESTS"
+              used: 700
```

Comment:
is this test technically over the limit, not nearing it?

Reply:
Oh, because of the `used` amount? I guess, but it doesn't really matter for the test. It's not like we do the math in the test runner.